corehq/apps/ota/urls.py

```diff
@@ -22,4 +23,5 @@
     url(r'^heartbeat/(?P<app_build_id>[\w-]+)/$', heartbeat, name='phone_heartbeat'),
     url(r'^get_next_id/$', get_next_id, name='get_next_id'),
     url(r'^recovery_measures/(?P<build_id>[\w-]+)/$', recovery_measures, name='recovery_measures'),
+    url(r'^registry_case_details/$', registry_case_details, name='registry_case_details'),
```

Comment:
How about just `registry_case` so the URL doesn't look like it's tied to the case detail screen?